src/models/searchservice/response/directanswer/MultiLineTextFeaturedSnippetDirectAnswer.ts

```diff
@@ -0,0 +1,15 @@
+import { BuiltInFieldType } from './BuiltInFieldType';
+import { BaseFeaturedSnippetDirectAnswer } from './BaseFeaturedSnippetDirectAnswer';
+import { EnumOrLiteral } from '../../../utils/EnumOrLiteral';
+
+/**
+ * A {@link BaseFeaturedSnippetDirectAnswer} with 'multi_line_text' field type
+ *
+ * @public
+ */
+export interface MultiLineTextFeaturedSnippetDirectAnswer extends BaseFeaturedSnippetDirectAnswer<string> {
+  /** The value of the direct answer. */
```

Comment:
can this doc be inherited from the base direct answer interface?

Reply:
There's a remark in the base direct answer interface related to 'rich_text', that I didn't want since it's not applicable to multi line text field type, so I opted to have the doc here directly instead of inheriting it.